iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/sql/parser/AstBuilder.java

```diff
@@ -1190,6 +1191,14 @@ public Node visitRemoveDataNodeStatement(RelationalSqlParser.RemoveDataNodeState
     return new RemoveDataNode(nodeIds);
   }
 
+  @Override
+  public Node visitRemoveConfigNodeStatement(
+      RelationalSqlParser.RemoveConfigNodeStatementContext ctx) {
+    Integer nodeId =
+        Integer.parseInt(ctx.INTEGER_VALUE() == null ? "-1" : ctx.INTEGER_VALUE().getText());
```

Comment:
Can it be null? If so, why we do not judge this in line 1187?

Reply:
Thank you for your correction, it cannot be null